pillarbox-player/src/main/java/ch/srgssr/pillarbox/player/analytics/PlaybackSessionManager.kt

```diff
@@ -26,7 +26,7 @@ import kotlin.time.Duration.Companion.milliseconds
 /**
  * Manages playback sessions, representing interactions with individual [MediaItem]s.
  */
-class PlaybackSessionManager {
+class PlaybackSessionManager : AnalyticsListener {
```

Comment:
This means that now the Javadoc of this class will also contain every method of `AnalyticsListener`: https://android.pillarbox.ch/api/pillarbox-player/ch.srgssr.pillarbox.player.analytics/-playback-session-manager/index.html

Reply:
We can maybe try to [supress](https://kotlinlang.org/docs/kotlin-doc.html#suppress) those method from the doc

Reply:
or we can split inherited functions with `separateInheritedMembers` in the dokka configuration.